modules/ROOT/pages/_partials/data-sync/server-realtime-updates-amq.adoc

```diff
@@ -184,33 +217,53 @@ const client = mqtt.connect({
 
 const pubsub = new MQTTPubSub({ client })
 ----
-
-There are some additional options passed into `mqtt.connect`
++
+[NOTE]
+====
+The options for `mqtt.connect` are:
```

Comment:
The original text which said `There are some additional options passed into `mqtt.connect`` was in the context of enabling TLS for the specific use case of connecting to a broker in OpenShift, i.e. there are some additional options compared to the previous example where tls is not used.


The newer sentence `The options for mqtt.connect are:` is a bit inaccurate because there are a tonne of options for `mqtt.connect` documented here: https://www.npmjs.com/package/mqtt#mqttclientstreambuilder-options

Reply:
moved to it's own step and reworded